Expose collection metadata API documentation with Swagger (#1015)
OpenTermsArchive/engine: CHANGELOG.md

All changes that impact users of this module are documented in this file, in the [Common Changelog](https://common-changelog.org) format with some additional specifications defined in the CONTRIBUTING file. This codebase adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## Unreleased [minor]

_Full changeset and discussions: [#1015](https://github.com/OpenTermsArchive/engine/pull/1015)._

### Added

> - Expose collection metadata API documentation with a user-friendly interface

Reviewer: Let's avoid judgement (even positive) in our changelog.

```suggestion
- Embed [Swagger UI](https://swagger.io) for graphical user interface documentation of the API
```

> - Embed [Swagger UI](https://swagger.io) for graphical user interface documentation of the API

Reviewer: I believe we are missing some breaking changes announcements, don't we? 🙂

All changes that impact users of this module are documented in this file, in the [Common Changelog](https://common-changelog.org) format with some additional specifications defined in the CONTRIBUTING file. This codebase adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## Unreleased [minor]

_Full changeset and discussions: [#1015](https://github.com/OpenTermsArchive/engine/pull/1015)._

### Changed

- **Breaking:** Change path from `/services/:serviceId` to `/service/:serviceId` for direct access to a `service` resource; update paths accordingly in your own codebase
- **Breaking:** Change path from `/specs` to `/docs` for OpenAPI specification; update paths accordingly in your own codebase and set the HTTP header `Accept: application/json`

### Added

> - Embed [Swagger UI](https://swagger.io) for graphical user interface documentation of the API

Reviewer: ```suggestion
- Embed [Swagger UI](https://swagger.io) for graphical user interface documentation of the API; access it on `/docs`
```

> - Embed [Swagger UI](https://swagger.io) for graphical user interface documentation of the API

Reviewer: > Was the OpenAPI documentation already there before this changeset? 🤔 Didn't it move from `specs` to `docs`?

### Changed

> - Change path from `/services/:serviceId` to `/service/:serviceId` for direct access to a `service` resource; update paths accordingly in your own codebase

Reviewer: Always put breaking changes on top (before additions).
```suggestion
- **Breaking** Change path from `/services/:serviceId` to `/service/:serviceId` for direct access to a `service` resource; update paths accordingly in your own codebase
```
